test/tests/DeviceIdentificationTest.cpp

```diff
@@ -64,7 +64,37 @@ class DeviceIdentificationTestFixture : public ::testing::Test {
 
 TEST_F(DeviceIdentificationTestFixture, RegisteredMethods)
 {
+    EXPECT_CALL(service, ConfigLine())
```

Comment:
Now it is not my unittest but wouldn't it be better to just make one test out of this or if it is really desired to split it between a registered and a property test can me make helper funtions to reduce the duplicate code that now can be found in both tests to setup and tear down the mock code and plugin before the actual test is done? That will be much better for maintainability of the test code